samples/Thinktecture.Blazor.Sample/Program.cs

```diff
@@ -1,21 +1,30 @@
+using Fluxor;
 using Microsoft.AspNetCore.Components.Web;
 using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
 using Thinktecture.Blazor.AsyncClipboard;
 using Thinktecture.Blazor.Badging;
 using Thinktecture.Blazor.FileHandling;
 using Thinktecture.Blazor.PwaUpdate;
 using Thinktecture.Blazor.Sample;
+using Thinktecture.Blazor.ViewTransitions;
 using Thinktecture.Blazor.WebShare;
 
 var builder = WebAssemblyHostBuilder.CreateDefault(args);
 builder.RootComponents.Add<App>("#app");
 builder.RootComponents.Add<HeadOutlet>("head::after");
 
 builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
+builder.Services.AddFluxor(options =>

```

Comment:
This is only for sample data. Because it is much easier with fluxor instead to write the state handling by my self. If you don´t want a third party lib in the sample project, i will remove this and implement it by my self.